WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift

```diff
@@ -156,12 +167,40 @@ extension StoredCredentialsAuthenticator: LoginFacadeDelegate {
 
 @available(iOS 13, *)
 extension StoredCredentialsAuthenticator {
-    func presentLoginEpilogue(credentials: AuthenticatorCredentials) {
+    private func presentLoginEpilogue(credentials: AuthenticatorCredentials) {
         guard let navigationController = self.navigationController else {
             DDLogError("No navigation controller to present the login epilogue from")
             return
         }
         
         authenticationDelegate.presentLoginEpilogue(in: navigationController, for: credentials, onDismiss: {})
     }
+    
+    private func presentLoginEmailView(error: Error) {
```

Comment:
If these methods are meant for iCloud Keychain only, would you add some comments that mention it?

Reply:
Added.